.github/dependabot.yml

```
@@ -9,11 +9,8 @@ updates:
       interval: "weekly"
 
   - package-ecosystem: "github-actions"
-    directory: "/"
-    schedule:
-      interval: "weekly"
-
-  - package-ecosystem: "github-actions"
-    directory: "/.github/actions/check-ext-build"
+    directories:
+      - "/"
+      - "/.github/actions/*"
     schedule:
       interval: "weekly"
```

```suggestion
      interval: "weekly"
    groups:
      github-actions:
        patterns:
          - "*"
        update-types:
          - "minor"
          - "patch"
```

(I think we want to group those only, I took this from hxps-studio-services conf

---

fixed